kissfft.pc.in

```diff
@@ -5,5 +5,5 @@ Name: kissfft
 Description: A Fast Fourier Transform (FFT) library that tries to Keep it Simple, Stupid
 Version: @kissfft_VERSION@
 
-Libs: -L${libdir} -l@KISSFFT_OUTPUT_NAME@
-Cflags: -I${includedir} @PKG_KISSFFT_DEFS@
\ No newline at end of file
+Libs: @PKG_OPENMP@ -L${libdir} -l@KISSFFT_OUTPUT_NAME@
+Cflags: -I${includedir} @PKG_KISSFFT_DEFS@
```

Comment:
By convention `includedir` variable is the global one and `-I${includedir}/kissfft` would be used here. But I do not foresee any actual problems from not respecting the convention here.

Reply:
Good point! Let me fix that in both Makefiles and CMakeLists